app/models/mixins/miq_provision_quota_mixin.rb

```diff
@@ -247,42 +253,88 @@ def quota_find_prov_request_by_group(options)
 
   def quota_find_active_prov_request_by_owner(options)
     email = get_option(:owner_email).to_s.strip
-    quota_find_active_prov_request(options).select { |p| email.casecmp(p.get_option(:owner_email).to_s.strip) == 0 }
+    quota_find_active_prov_request(options).select { |p| email.casecmp(p.request_owner_email(p)).zero? }
+  end
+
+  def request_owner_email(request)
+    service_request?(request) ? request.requester.email : request.get_option(:owner_email).to_s.strip
+  end
+
+  def service_request?(request)
+    request.type == "ServiceTemplateProvisionRequest"
```

Comment:
Suggest `request.kind_of?(ServiceTemplateProvisionRequest)`.

This will future-proof the method if the ServiceTemplateProvisionRequest ever gets sub-classed.  Because the name isn't already long enough.  😄